state/indexer/block/kv/kv_test.go

```diff
@@ -1,23 +1,96 @@
-package kv_test
+package kv
```

Comment:
```suggestion
package kv_test
```

we cannot change package names.